plugins/pluginmanagers/lz-n.nix

```diff
@@ -0,0 +1,200 @@
+{
+  lib,
+  options,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+let
+  inherit (lib.nixvim) defaultNullOpts;
```

Comment:
Good idea! If we're gonna inherit `defaultNullOpts`, maybe also add `inherit (lib) types`? I think that's pretty much the only thing that is currently justifying the `with lib`?

Not important, just a suggestion. Feel free to ignore or wait for feedback from the other maintainers.